File: pi/src/entity/Demande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package entity;
import java.sql.Date;


/**
 *
 * @author ABS1
 */
public class Demande {
    private int id;
   private  String date;
    private String  nomterrain;
    private String nomequipe;
    private String email;
    private String etat;

    public String getEtat() {
        return etat;
    }

    public Demande(String date, String nomterrain, String nomequipe, String email, String etat) {
        this.date = date;
        this.nomterrain = nomterrain;
        this.nomequipe = nomequipe;
        this.email = email;
        this.etat = etat;
    }

    public Demande(int id, String date, String nomterrain, String nomequipe, String email, String etat) {
        this.id = id;
        this.date = date;
        this.nomterrain = nomterrain;
        this.nomequipe = nomequipe;
        this.email = email;
        this.etat = etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public Demande(String date, String nomterrain, String nomequipe, String email) {
        this.date = date;
        this.nomterrain = nomterrain;
        this.nomequipe = nomequipe;
        this.email = email;
    }
    
    public Demande (int id, String date, String nomterrain, String nomequipe,String email) {
        this.id = id;
        this.date =date;
        this.nomterrain = nomterrain;
        this.nomequipe= nomequipe;
        this.email=email;
        
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getEmail() {
        return email;
    }

    public Demande(int id) {
        this.id = id;
    }
    

    public Demande() {
    }

    public Demande(int id, String nomterrain, String nomequipe) {
        this.id = id;
        this.nomterrain = nomterrain;
        this.nomequipe = nomequipe;
    }

    
    

    public Demande(String date, String nomterrain, String nomequipe) {
        this.date = date;
        this.nomterrain = nomterrain;
        this.nomequipe = nomequipe;
    }

    public int getId() {
        return id;
    }

    public String getDate() {
        return date;
    }

    public String getNomterrain() {
        return nomterrain;
    }

    public String getNomequipe() {
        return nomequipe;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public void setNomterrain(String nomterrain) {
        this.nomterrain = nomterrain;
    }

    public void setNomequipe(String nomequipe) {
        this.nomequipe = nomequipe;
    }

    @Override
    public String toString() {
        return "Demande{" + "id=" + id + ", date=" + date + ", nomterrain=" + nomterrain + ", nomequipe=" + nomequipe + '}';
    }
    
    
    
     
}

File: src/Entity/Demande.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Demande
 *
 * @ORM\Table(name="demande")
 * @ORM\Entity
 * @ORM\Entity(repositoryClass="App\Repository\DemandeRepository")
 */
class Demande
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="date", type="string", length=200, nullable=false)
     */
    private $date;

    /**
     * @var string
     *
     * @ORM\Column(name="nomterrain", type="string", length=200, nullable=false)
     */
    private $nomterrain;

    /**
     * @var string
     *
     * @ORM\Column(name="nomequipe", type="string", length=200, nullable=false)
     */
    private $nomequipe;

    /**
     * @var string
     *
     * @ORM\Column(name="email", type="string", length=200, nullable=false)
     */
    private $email;

    /**
     * @var string
     *
     * @ORM\Column(name="etat", type="string", length=200, nullable=false)
     */
    private $etat;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDate(): ?string
    {
        return $this->date;
    }

    public function setDate(string $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getNomterrain(): ?string
    {
        return $this->nomterrain;
    }

    public function setNomterrain(string $nomterrain): self
    {
        $this->nomterrain = $nomterrain;

        return $this;
    }

    public function getNomequipe(): ?string
    {
        return $this->nomequipe;
    }

    public function setNomequipe(string $nomequipe): self
    {
        $this->nomequipe = $nomequipe;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }


}
